src/tribler-core/tribler_core/modules/ipv8_module_catalog.py

```diff
@@ -185,7 +185,9 @@ class PopularityCommunityLauncher(IPv8CommunityLauncher):
 @walk_strategy(random_walk, target_peers=30)
 @walk_strategy(remove_peers, target_peers=INFINITE)
 class GigaChannelCommunityLauncher(IPv8CommunityLauncher):
-    pass
+    def get_kwargs(self, session):
+        from tribler_core.modules.metadata_store.community.discovery_booster import DiscoveryBooster
+        return {'components': [DiscoveryBooster()]}
```

Comment:
This way of running subcomponents is not standard to IPv8. If there would be more than a single subsystem running like this, this would be the correct way to do it (however, it would be even more correct to add the "components" thing into IPv8 `Community` class). 

But you're running only a single component like this. Inventing a separate subsystem to run just a single component is a form of overengineering. Of course, this sends the other programmers a signal: "do it like this next time". However, that's not the point of this PR. Putting the thing here evoked a "WTF?" moment from me. So, please just put the thing into the GigaChannelCommunity `_init_`.

Reply:
@ichorid proposed approach will break our test system because I calling 

```python
    def take_step(self):
        self.logger.debug('Take a step')
        self.walker.take_step()
```
https://github.com/drew2a/tribler/blob/e54e3f080dae3278e9ca29695774959fef3378ad/src/tribler-core/tribler_core/modules/metadata_store/community/discovery_booster.py#L52

Reply:
Why not `patch` it then?

Reply:
What do you mean?

Reply:
Like this:
https://github.com/drew2a/tribler/compare/feature/5828...ichorid:feature/5828

Reply:
👍 thank you